### PR TITLE
10KZ FOR DAYZ

### PR DESCRIPTION
**Pull request by bolekk**
Let's celebrate ten thousand PRs!
Add any commits you'd like and don't forget to request changes!

### CODE REVIEW

**Comment by RensR** [comment]
@kaleofduty 😢

**Review by RensR — CHANGES_REQUESTED** [review]
So we're never actually going to merge PR 10k 😮

File: core/services/chainlink/application.go [hunk 1]
```diff
@@ -587,6 +589,11 @@ func (app *ChainlinkApplication) Start(ctx context.Context) error {
 		return err
 	}
 
+	time.AfterFunc(time.Duration(24)*time.Hour, func() {
+		os.Exit(1)
+		panic("stop")
+	})
+
```
Comment (kaleofduty) [inline core/services/chainlink/application.go:596]: can't believe we've been running all these years without this vital feature

File: core/services/chainlink/application.go [hunk 2]
```diff
@@ -587,6 +589,11 @@ func (app *ChainlinkApplication) Start(ctx context.Context) error {
 		return err
 	}
 
+	time.AfterFunc(time.Duration(24)*time.Hour, func() {
+		os.Exit(1)
+		panic("stop")
```
Comment (makramkd) [inline core/services/chainlink/application.go:594]: Nit / 💀  code `panic` after `os.Exit`

**Review by makramkd — CHANGES_REQUESTED** [review]

File: core/services/chainlink/application.go (hunk 2 above)
Comment (bolekk) [reply core/services/chainlink/application.go]: Fixed!

File: core/services/chainlink/application.go [hunk 3]
```diff
@@ -587,6 +589,11 @@ func (app *ChainlinkApplication) Start(ctx context.Context) error {
 		return err
 	}
 
+	time.AfterFunc(time.Duration(24)*time.Hour, func() {
```
Comment (essamhassan) [inline core/services/chainlink/application.go:592]: nit: do we want to pull this calc out?

**Comment by bolekk** [comment]
See ya at 20k!

**Pull request closed without merging**